### PR TITLE
MDEXP-333 Bugfix release 3.0.2

### PR DESCRIPTION
**Pull request by andrii-novytskyi**
https://issues.folio.org/browse/MDEXP-333
## Purpose
This is a bugfix release for correcting SQL statement name in the update_transformations_mapping_profile migration script.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
+## 11/09/2020 v2.1.3 Released
```
Comment (KVupp) [inline NEWS.md:1]:
```suggestion
## 11/09/2020 v3.0.2 Released
```

File: src/main/resources/templates/db_scripts/schema.json [hunk 2]
```diff
@@ -104,6 +104,16 @@
       "run": "after",
       "snippetPath": "migration_scripts/add_last_updated_date_to_job.sql",
       "fromModuleVersion": "mod-data-export-2.2.0"
+    },
+    {
+      "run": "after",
+      "snippetPath": "migration_scripts/remove_hasSameTagInHoldings.sql",
+      "fromModuleVersion": "mod-data-export-3.0.1"
```
Comment (KVupp) [inline src/main/resources/templates/db_scripts/schema.json:111]: I think we can make this 3.0.2 as well

**Review by KVupp — APPROVED** [review]

File: pom.xml [hunk 3]
```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.folio</groupId>
   <artifactId>mod-data-export</artifactId>
-  <version>3.1.0-SNAPSHOT</version>
+  <version>3.0.1</version>
```
Comment (KVupp) [inline pom.xml:6]: change this to 3.1.0-SNAPSHOT, other wise the release plugin will not work as it is not a snapshot version

File: pom.xml [hunk 4]
```diff
@@ -513,7 +506,7 @@
     <url>https://github.com/folio-org/mod-data-export</url>
     <connection>scm:git:git://github.com/folio-org/mod-data-export</connection>
     <developerConnection>scm:git:git@github.com:folio-org/mod-data-export.git</developerConnection>
-    <tag>HEAD</tag>
+    <tag>v3.0.1</tag>
```
Comment (KVupp) [inline pom.xml:509]: same here revert to the HEAD tag

**Comment by andrii-novytskyi** [comment]
Release changes merged in this PR - https://github.com/folio-org/mod-data-export/pull/201

**Pull request closed without merging**